src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4832nocstylearray/InputArrayTypeStyle.java

```diff
@@ -29,14 +29,19 @@ public static void mainC(String aCStyle[]) //warn
 
         public Test[]
             getTests()
-        { // we shouldn't check methods because there is no alternatives.
+        {
             return null;
         }
 
         public Test[] getNewTest() //ok
         {
             return null;
         }
+
+        public Test getOldTest()[] //warn
```

Comment:
Please add cases for multi dimensional arrays like `getOldTest()[][]`

Reply:
I also don't see any examples like `Test[][] getNewTest()`. Also add one like `Test[] getNewTest()[]`.